posts/2023-02-21-23.0.0.2-beta.adoc

```diff
@@ -0,0 +1,389 @@
+open-graph-image: https://openliberty.io/img/blog/ac_db_test_container_auth.png
+open-graph-image-alt: Open Liberty Logo
+---
+= Testing database connections and Jakarta Data preview in Open Liberty 23.0.0.2-beta
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+Open Liberty 23.0.0.2-beta includes several new enhancements and changes for Liberty InstantOn, aimed at simplifying the process for creating checkpoints and restoring them.  Admin Center's Server Config tool receives new capability to test database connections, and a very early preview of Jakarta Data also makes its beta debut in this release.  Also included in this beta is all of Jakarta EE Platform 10 and MicroProfile 6.
+
+
+The link:{url-about}[Open Liberty] 23.0.0.2-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+* <<instanton, New enhancements for Liberty InstantOn>>
+* <<validation, Testing database connections using Admin Center>>
+* <<data, A very early preview of Jakarta Data>>
+* <<ee10, Jakarta EE Platform 10 and MicroProfile 10>>
+
+// // // // // // // //
+// In the preceding section:
+// Change SUB_FEATURE_TITLE to the feature that is included in this release and
+// change the SUB_TAG_1/2/3 to the heading tags
+//
+// However if there's only 1 new feature, delete the previous section and change it to the following sentence:
+// "The link:{url-about}[Open Liberty] 23.0.0.2-beta includes SUB_FEATURE_TITLE"
+// // // // // // // //
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+[#instanton]
+== New enhancements for Liberty InstantOn
+The Open Liberty 22.0.0.11-beta introduced InstantOn, an exciting new feature that provides incredibly fast startup times for MicroProfile and Jakarta EE applications. Since the initial beta release, we've made a few changes and enhancements to make it easier to create and deploy applications with Liberty InstantOn.  They are as follows: 
+
+* Removal of the features checkpoint phase 
+* Reduced set of required Linux capabilities to checkpoint and restore
+* Simplified container image builds with InstantOn
+
+To learn more about these changes and InstantOn, refer to the link:{url-prefix}/blog/2023/02/10/instant-on-beta-update.html[New enhancements for Liberty InstantOn in 23.0.0.2-beta] blog post.
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/23629
+// Contact/Reviewer: ReeceNana,k8vance88
+// // // // // // // // 
+[#validation]
+== Testing database connections in using Admin Center   

```

Comment:
This is missing a word.  Should it be "... in Open Liberty using Admin Center" ?

Reply:
```suggestion
== Testing database connections using Admin Center   

```